site/content/en/guides/consumer/update/_index.md

```diff
@@ -166,6 +166,8 @@ kpt will throw an error if trying to update a package and the git repo
 has uncommitted changes.
 {{% /pageinfo %}}
 
+Move into the directory where you cloned the repo (helloworld in thie example)
```

Comment:
I suggest the following flow for this document:

```
mkdir workspace
git init
kpt pkg get $REPO/package-examples/helloworld-set@v0.3.0 helloworld
git add .
git commit -m "init"
// Edit helloworld
git add .
git commit -m "mychanges"
kpt pkg update ...
```

In general, we want all terminal commands to be explicit.

If this is more than what you signed up for, I'll follow up with a PR making these changes.

Reply:
Done, PTAL